lightning/src/ln/channelmanager.rs

```diff
@@ -416,10 +416,10 @@ pub(crate) enum PendingOutboundPayment {
 		/// Our best known block height at the time this payment was initiated.
 		starting_block_height: u32,
 	},
-	/// When a payment completes, we continue tracking it until all pendings HTLCs have been
-	/// resolved. This ensures we don't look up pending payments in ChannelMonitors on restart and
-	/// add a pending payment that was already completed.
-	Completed {
+	/// When a pending payment completes, we continue tracking it until all pendings HTLCs have
+	/// been resolved. This ensures we don't look up pending payments in ChannelMonitors on restart
+	/// and add a pending payment that was already completed.
+	Fulfilled {
```

Comment:
Terminology question related to my earlier comment: is there a difference between "completes" and "fulfilled" as it pertains to the first sentence in the docs and the variant name?

Reply:
No, I just missed the docs in the rename - once we get one HTLC fulfillment we consider the payment fulfilled.